rust/integration-tests/src/sdk_tests/sdk_tests.rs

```diff
@@ -0,0 +1,134 @@
+#[cfg(test)]
+mod tests {
+    use std::fs;
```

Comment:
@rtso @dermanyang probably only file worth taking a look for now

Reply:
are we planing on adding all tests into this file?

Reply:
no there will be tests for sdk event processor only. renamed to events_processor_tests